example_school_library_decorator/rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
```

Comment:
- [ ] A rental belongs to a book and also belongs to a person. Currently, you only have a rental-book association. Kindly add the `Getters and Setters`(attr_accessor) for a person then initialize in the constructor and also a rental-person` belongs to` association.